src/index.js

```diff
@@ -368,11 +373,13 @@ const generateOwnershipWallet = async config => {
   const buf = Buffer.from(ob.patq2hex(ticket), 'hex')
   const masterSeed = await argon2u(buf, ship)
 
-  return deriveNode(
+  const node = deriveNode(
     masterSeed,
     CHILD_SEED_TYPES.OWNERSHIP,
     passphrase
   );
+
+  return { type: CHILD_SEED_TYPES.OWNERSHIP, ...node }
```

Comment:
Shouldn't the `type` just be returned from `deriveNode` instead?

Reply:
yes, good catch.